mycpp/TEST.sh

```diff
@@ -257,12 +257,15 @@ test-control-flow-graph() {
     banner "$ex"
 
     translate-example $ex
-    diff -u $data_dir/cf_edge.facts _tmp/mycpp-facts/cf_edge.facts
+    for fact_path in $data_dir/*.facts; do
+      local fact_file=$(basename $fact_path)
+      diff -u $data_dir/$fact_file _tmp/mycpp-facts/$fact_file
+    done
   done
 }
 
 golden-control-flow-graph() {
-  echo 'TODO: regenerate data in testdata/control-flow-graph'
+  run-test-func test-control-flow-graph _test/mycpp/test-cfg-examples.log
```

Comment:
I think we should delete this "golden" function and just uncomment the original line

Now I see you added the `testdata/control-flow-graph.sh` script that does what this was supposed to